react/features/base/lastn/middleware.js

```diff
@@ -92,6 +96,7 @@ MiddlewareRegistry.register(store => next => action => {
     switch (action.type) {
     case APP_STATE_CHANGED:
     case CONFERENCE_JOINED:
+    case FAKE_SCREEN_SHARE_REMOTE_PARTICIPANTS_UPDATED:
```

Comment:
What's the part of `_updateLastN` logic that's affected by `FAKE_SCREEN_SHARE_REMOTE_PARTICIPANTS_UPDATED`?

Reply:
`FAKE_SCREEN_SHARE_REMOTE_PARTICIPANTS_UPDATED` it still used to set `remoteScreenShares` used in `_updateLastN`

In the old flow we have:
`state['features/video-layout'].remoteScreenShares` and  `state['features/base/participants'].sortedRemoteParticipants`
In the new flow:
`state['features/video-layout'].remoteScreenShares` and  `state['features/base/participants'].sortedFakeScreenShareParticipants`